### ISSUE TITLE
Fast Shooting Projectiles

### ISSUE THREAD

**Issue by andwhyisit** [andwn/cave-story-md#133]
The Armor Gaudi projectiles in the Boulder to Core run of the Labyrinth, and the Dragon Zombie fire projectiles in the Egg Corridor both can travel at crazy one-screen-per-second speeds.

**Comment by andwhyisit** [comment]
Uhh..

The Gaudi are aiming to their upper-left when you are to the left of them and to the bottom-left when you are to the right of them. They should aim at the player.

**Comment by andwhyisit** [comment]
The Gaudi still aren't aiming at the player. They shoot super fast off to the right in a straight direction when the player is to the right of them, or at normal speed towards the upper left if the player is to the left.

**Comment by andwn** [comment]
The Gaudis are fine here. Could you check the build log for the nightly? Maybe one of the SGDK tools has a dependency I didn't notice.

**Comment by andwn** [comment]
Yep I definitely broke the nightly. Some nuance with the linker or make clean probably.

**Comment by andwn** [comment]
The nightly doesn't 404 anymore so I solved the mystery? It was one of these 2 things:

1. Native GCC was too old for -std=gnu11. I changed these to either c99 or gnu99. Hope you like warnings.

2. Gendev GCC had left over libgcc 4.9.x from the old version before it was rebuilt with 6.3.0. This library is in a specific directory that is named after the GCC version. My makefile was using a wildcard for that directory to link with libgcc, so it may have been trying to grab the wrong version. Now it instead detects Gendev's GCC version for the directory name.
